### DISCUSSION TITLE
[General] Where to get postgres dump to run external node?

### DISCUSSION THREAD

**Discussion by sajal** [zkSync-Community-Hub/zksync-developers#108] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I see from [the docs](https://github.com/matter-labs/zksync-era/blob/main/docs/external-node/03_running.md), that one is expected to have a "DB dump" in order to run External Node.

Where would one get such a dump? 

This feels like a permissioned chain if only insiders can run a full node.

### What is the specific issue or error you're encountering?

inability to run full node.

### Can you share the error messages or logs you're receiving, if any?

none

### Have you made any recent changes to the contract before encountering this issue?

none

### Are there any external libraries or contracts that your contract interacts with?

none

### Can you provide the relevant portions of your contract code where the issue is occurring?

none

### Have you tried to isolate the problem, and if so, what were the results?

none

### What steps have you already taken to try to resolve the issue?

Tried joining zksync discord to ask, desnt work

**Comment by githubdoramon** [comment, marked as answer]
@sajal the snapshot dump is not yet public. The External Node alpha release is intended for some initial partners to make sure it is operating properly. We are planning on the next steps to make the public release and then the snapshot will also be public.

**Reply by sajal** [reply]
Could you make it clear in [the doc](https://github.com/matter-labs/zksync-era/blob/main/docs/external-node/03_running.md) that this is only for "partners".

**Reply by githubdoramon** [reply]
More than fair request. Will do it @sajal . Thanks

**Reply by arose0** [reply]
@sajal also, just FYI, this snapshot isn't required because some of the data is held privately - it is just that currently syncing from genesis is slow and we want people to have the option to accelerate this by starting from the snapshot. 

The plan is to provide these publicly ASAP too.

**Comment by hatemosphere** [comment]
It is also worth emphasizing it's not a **full** node, as we are still operating on a centralized sequencer, please find more details [in docs](https://github.com/matter-labs/zksync-era/blob/main/docs/external-node/01_intro.md#what-is-the-external-node) However, we are actively working on decentralizing various components, so please stay tuned.

**Reply by sajal** [reply]
I get that its not a "full" node, for a lot of people, having their own copy of their blockchain node that they can run queries against (without rate limits or network overhead) is very useful, it seems that right now only "initial partners" are granted that privilege. The documentation is misleading and it leads people to believe that anyone can run the external node.

**Reply by benjichat** [reply]
Two questions:

- Is it possible to be included in the initial partners set?

- Our team would like to run it's own sequencer. Is this possible?

**Reply by benjichat** [reply]
@githubdoramon see above questions

**Reply by benjichat** [reply]
@hatemosphere how do we get in touch about being part of the partners set and helping with decentralizing various components?

**Comment by daisbryce** [comment]
The "DB dump" required to run an External Node in zkSync is provided by the zkSync team upon request. It contains the necessary data to initialize and synchronize the node with the zkSync network. To obtain this dump, you can reach out to the zkSync team through their official channels, such as Discord or their website. Once you have the dump, you can proceed with setting up and running your External Node.

**Comment by etesvipatdeb** [comment]
Certainly! Obtaining a "DB dump" for running an External Node on Mainnet typically requires access to the zkSync team's infrastructure. This process ensures the security and integrity of the network data. You may need to reach out to the zkSync team directly or follow their documentation for guidance on obtaining the necessary resources to run a full node.

**Comment by poinsondrew** [comment]
To run a ZkSync External Node, you need to obtain a database dump. You can request access to the dump from the ZkSync team through their official channels, such as their Discord server or support email. This ensures the integrity and security of the network while allowing authorized users to operate full nodes.

**Comment by laducerbecky** [comment]
It appears that in order to run an External Node on the Mainnet, one is expected to have a "DB dump," but it is unclear where one can obtain such a dump. This requirement may suggest that the Mainnet is permissioned and only insiders have access to the necessary resources to run a full node. The user has attempted to seek help on the zksync Discord channel but has not received a response.

**Comment by hilliardloyd** [comment]
In order to run an External Node on Mainnet, you would need to obtain a DB dump from the zkSync team. This is because running a full node requires a significant amount of resources and is not feasible for most individual users. The DB dump contains a snapshot of the blockchain at a specific point in time, allowing you to sync up to the current state of the network. However, access to the DB dump is currently restricted to zkSync team members and select partners.
